mols2grid/molgrid.py

```diff
@@ -198,7 +199,12 @@ def __init__(  # noqa: PLR0912
                 register._update_current_grid(name)
         else:
             self._cached_selection = {}
-            register._init_grid(name)
+            if is_running_within_marimo():
```

Comment:
it would be better to directly skip emitting the warning in `_init_grid` [here](https://github.com/cbouy/mols2grid/blob/master/mols2grid/select.py#L24):
```python
if overwrite and not is_running_within_marimo():
```